sdk/CMakeLists.txt

```diff
@@ -50,15 +50,15 @@ set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
 set(CMAKE_INSTALL_IPATH ${CMAKE_INSTALL_PREFIX}/include)
 
 aux_source_directory(${MCMDP_SRC_DIR} sdk_srcs)
-add_library(mcm_dp SHARED ${sdk_srcs})
```

Comment:
Why this change? I though that the idea is to create a dynamic library and .so file of SDK which a user can load. The header files, SDK .so, and already compiled media_proxy is all what they need. If I'm not wrong after the change from SHARED to OBJECT the .so wont be created, so a user can't easily link their code with the SDK.

Reply:
ok, now I see it. You then create a shared lib with: `add_library(mcm_dp_shared SHARED $<TARGET_OBJECTS:mcm_dp>)`. Never mind.